index.html

```diff
@@ -1,61 +1,144 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"

```

Comment:
please dont use a cdn for this. Just at npm and add bootstrap as an dependency.

Reply:
The base of the project is only a static HTML page. Convert it to a node js project kind of headache and not necessary. And all the cdn are cacheable so it is already available in the browser We don't need a get request for fetching pooper js or bootstrap css file.